code/src/components/MovieDetails.js

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from "react";
+import { useParams, useNavigate } from "react-router-dom";
+
+import arrow from "images/arrow.png"
+import { SINGLE_MOVIE_URL } from "utils/urls";
+import { 
+  Poster, 
+  MovieDetailsContainer, 
+  MovieDetailsWrapper, 
+  MovieDetail, 
+  Rating, 
+  MovieDescription, 
+  BackButton,
+  BackArrow,
+  ButtonText
+} from "styles";
+
+const MovieDetails = () => {
+  const [movies, setMovies] = useState({});
+  const { movieId } = useParams();
+  const navigate = useNavigate();
+
+  const onBackButtonClick = () => {
+    navigate(-1);
+  };
+
+  useEffect(() => {
+    fetch(SINGLE_MOVIE_URL(movieId))
+      .then((res) => res.json())
+      .then((data) => setMovies(data))
+      .catch(error => console.error(error))
+  }, [movieId]);
+
+  return (
+    <MovieDetailsContainer url={movies.backdrop_path ? `url(https://image.tmdb.org/t/p/w1280${movies.backdrop_path})` : ''} >
+      <BackButton onClick={onBackButtonClick}>
+        <BackArrow src={arrow} alt="back arrow icon"></BackArrow>
```

Comment:
Not completely following here (new to styled components) but do I understand it correctly - you are importing and rendering "components" (that are, not strictly speaking components) from your styles.js?